app/uk/gov/hmrc/eoricommoncomponent/frontend/views/helpers/inputDate.scala.html

```diff
@@ -97,6 +99,7 @@ <h1 class="heading-large">
             '_labelClass -> "form-group form-group-year",
             '_inputClass -> inputClass(yearField),
             '_pattern -> "[0-9]*",
```

Comment:
Is this pattern not an issue? I remember that having pattern will add this pop up error for the input.

Reply:
That is fixed as part of ECC-327